addons/l10n_ec/models/account_tax_group.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+_TYPE_EC = [
+    ("vat12", "VAT 12%"),

```

Comment:
not that important, but is not it better to use vat and derive the percentage from the amount on the tax?

Reply:
There's another taxes with more complex implementation, on withholdings we have various amounts, to get only one method prefer this form